docs/wip/data-fetching.md

```diff
@@ -1,8 +1,45 @@
+
 ### Static data
+
+Static data is the content that does not change frequently or per user interaction. For example, a product's page can be a data retrieve static since the description and attributes of product don't change very often. This means that this data can be fetch during build time and when the user access the products page the data is already loaded.
+
+There is two ways to fetch static data in Gatsby: [Page Query](https://www.gatsbyjs.com/docs/recipes/querying-data#querying-data-with-a-page-query) and [Static Query](https://www.gatsbyjs.com/docs/how-to/querying-data/static-query/#reach-skip-nav).
+
 ### Dynamic data
 
-### What we provided out of the box with `gatsby-source-vtex`
+If the data constantly changes or depends on the user's interaction, it is a strong indication that it must be dynamic. For example, the shipping calculation changes depending on the location of the user who is interacting or other example is the implementation of a shopping cart that changes its data every time a product is added, being necessary to communicate with the backend of the application every time that the user interacts with the cart.
+
+For fetching data at runtime in the browser, you can use any method to retrieve data that you would use in a regular React app, such as using the `fetch` API for retrivieng data of a Rest API or `apollo-client` to retrieve from a GraphQL API.
+
+## What we provided out of the box with `gatsby-source-vtex`
+
+`gatsby-source-vtex` plugin connects the [Gatsby data layer]((https://www.gatsbyjs.com/docs/porting-from-create-react-app-to-gatsby/#unified-graphql-data-layer)) to [VTEX's GraphQL](https://github.com/vtex-apps/store-graphql) allowing to access VTEX API through your Gatsby application.
+
+For the plugin to work it is necessary in the `gatsby-config.js` file to change the `STORE_ID` variable to the desired VTEX account. It is also possible to edit the variables `enviroment` and` workspace`, example:
+
+```js
+plugins: [
+    {
+      resolve: `@vtex/gatsby-source-vtex`,
+      options: {
+        tenant: STORE_ID, // VTEX's account
+        environment: 'vtexcommercestable' , // it can be vtexcommercestable or vtexcommercebeta
+        workspace: 'master', // https://vtex.io/docs/concepts/workspace/ 
+      },
+    },
+```
+
+If all goes well, when running the site with the command `yarn develop` the output at the end will contain the URL to access Gatsby's Graphql playground:
+
+```
+View GraphiQL, an in-browser IDE, to explore your site data and schema
+⠀
+http://localhost:8000/___graphql
+```
+
+Access this URL and you will see in the Schema all queries and mutations of the VTEX API, feel free to test the queries and see what they return. To use this data on your website, just follow what is indicated in the previous topic of [How to fetch data for your pages] (# How-to-fetch-data-for-your-pages).
```

Comment:
Access this URL and to see all queries and mutations of the VTEX API in the Schema. 
 
Feel free to test the queries and see what they return. To use this data on your website, just follow what is indicated in the previous topic of [How to fetch data for your pages] (# How-to-fetch-data-for-your-pages).